Makefile

```diff
@@ -25,6 +25,9 @@ test:
 	-docker-compose build housing-finance-interim-api-test && docker-compose run housing-finance-interim-api-test
 	-make clean
 
+test-db:
+	dotnet test --filter FullyQualifiedName~Infrastructure.DatabaseContext
+
```

Comment:
They're excluded from CI at the moment, so they're somewhat logically separated and it's easier to run them separately.

It shouldn't be too difficult to add them to CI, but that can be a separate PR, and this can be removed at that point. Or I can remove it now, I don't mind.